fearless/Modules/Wallet/InvoiceScan/InvoiceScanLocalSearchEngine.swift

```diff
@@ -11,21 +11,22 @@ final class InvoiceScanLocalSearchEngine: InvoiceLocalSearchEngineProtocol {
         self.networkType = networkType
     }
 
-    func searchByAccountId(_ accountId: String) -> SearchData? {
-        guard let accountIdData = try? Data(hexString: accountId),
-              accountIdData.count == ExtrinsicConstants.accountIdLength
-        else {
+    func searchByAccountId(_ accountIdHex: String) -> SearchData? {
+        guard let accountId = AccountId.matchHex(accountIdHex) else {
```

Comment:
It's not obvious what this function does. Do you think it is possible to rename it to make code more readable? Unfortunately, I myself don't have any good suggestions...